src/mme-app/Makefile

```diff
@@ -48,6 +49,9 @@ LIBS :=	-lpthread \
         -lgrpc \
         -lgpr \
         -lprotobuf \
+	-lepc \
+	-lcares	\
```

Comment:
just for my information...what are these libraries ?

Reply:
These are related to epctool library